src/06-excitation-transfer-revisited.md

```diff
@@ -0,0 +1,696 @@
+
+There is however still some significant dependency on $\Delta n$ that's not being captured by the $\Delta E$ scaling we've found. In other words, the lines aren't horizontal. In the spirit of connecting to our earlier scalings with 2 matched TLS, we can try adding the coupling:  
+
+
+$$T \sim \left(\frac{\Delta E}{U}\right)^{2+\Delta n}$$
+
+Let's check this.
+
+```python
+scaled_transfer_times_df = transfer_times_df.copy()
+
+for index, row in scaled_transfer_times_df.iterrows():
+    for DeltaE2 in DeltaE2s:
+        scaled_transfer_times_df.loc[index, DeltaE2] /= ((DeltaE2+row["bosons"]+DeltaE2)/2/(np.sqrt(1)*U))**(2+row["bosons"])
+```
+
+```python
+ax = scaled_transfer_times_df.plot(x="bosons", logy = True, title=f"Scaled Excitation transfer time for mismatched TLS with  \n {H_latex} \n $\omega=1$, $U=0.1$ (Fig. 15)", legend=True, style='o-');
+ax.set_xlabel("$\Delta E_1 - \Delta E_2$ (Bosons emitted $\Delta n$)");
+ax.set_ylabel("$\log(T)$");
+plt.annotate('$\Delta E_2$', xy=(0.15, 1.2*10**3));
+```
+
+Looks like we've overcompensated in Fig. 15. At this point, we could start getting more systematic to find the best coefficients in our model. If we look again at the 2008 work of [Hagelstein](http://dx.doi.org/10.1088/0953-4075/41/13/135501), we find that all we're missing is a $\sqrt{8}$:
+
+$$T \sim \left(\frac{\Delta E}{U\sqrt{8}}\right)^{2+\Delta n}$$
+
+
+```python
+scaled_transfer_times_df = transfer_times_df.copy()
+
+for index, row in scaled_transfer_times_df.iterrows():
+    for DeltaE2 in DeltaE2s:
+        scaled_transfer_times_df.loc[index, DeltaE2] /= ((DeltaE2+row["bosons"]+DeltaE2)/2/(np.sqrt(8)*U))**(2+row["bosons"])
+```
+
+```python
+ax = scaled_transfer_times_df.plot(x="bosons", logy = True, title=f"Scaled Excitation transfer time for mismatched TLS with  \n {H_latex} \n $\omega=1$, $U=0.1$ (Fig. 16)", legend=True, style='o-');
+ax.set_xlabel("$\Delta E_1 - \Delta E_2$ (Bosons emitted $\Delta n$)");
+ax.set_ylabel("$\log(T)$");
+plt.annotate('$\Delta E_2$', xy=(0.15, 1.2*10**3));
+```
+
+Of the 14 orders of magnitude difference that we saw in Fig 13, our model for the scaling of excitation transfer time explains 13 of them. This is not bad considering that the analytical work of Hagelstein works on the assumption of a highly excited oscillator (i.e, lots of bosons already present in the system) which we didn't have in our simulations.
+
+
+What can we take away from the scalings we've found for excitation transfer time? If we don't have identical systems, then it's going to be very hard for us to observe excitation transfer due to the exponential scaling with TLS transition energy mismatch - it will just be too slow. Even with identical systems, the quadratic scaling with $\Delta E / U$ is also likely to make observing excitation transfer difficult because typically $\Delta E \gg U$.
```

Comment:
I'm still lacking a bit of intuition as to how (even if tiny) much of a deviation from the resonance line may be tolerable in some cases. No need to address this in the notebook but let's discuss this more

Reply:
Agreed.

I guess we'd need to look at comparing the excitation transfer time to some typical decoherence time? 

The formula 
$$T \sim \left(\frac{\Delta E}{U\sqrt{8}}\right)^{2+\Delta n}$$

is useful for calculating how much longer the excitation transfer time will be when we're mismatched ($\Delta n \neq 0$) compared to the case when we are matched ($\Delta n = 0$).  It will scale like

$$ \left(\frac{\Delta E}{U\sqrt{8}}\right)^{\Delta n}$$

So we might then look at the smallest mismatch of $\Delta n=2$ (it's got to be even because of parity) and see how big that is. If we then had an actual physical number for the matched case then we could say something I think.

Reply:
Ah right right, this is very helpful. I think ultimately it would be helpful to give an example so it is less abstract. But I'm not sure whether here is the right place, or we can come back to this at a later point and add a specific example with concrete values. This is all great for now I think.